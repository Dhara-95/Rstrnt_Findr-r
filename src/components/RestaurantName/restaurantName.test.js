import React from "react";
import renderer from "react-test-renderer";
import { RestaurantName } from "./index";

it("displays the Restaurant name correctly", () => {
  const tree = renderer.create(<RestaurantName name={"name"} />).toJSON();
  expect(tree).toMatchSnapshot();
});
