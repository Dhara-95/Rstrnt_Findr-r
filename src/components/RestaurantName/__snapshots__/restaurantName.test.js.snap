// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the Restaurant name correctly 1`] = `
<h2
  className="text-grey text-xl"
>
  name
</h2>
`;
